keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/DeliveryArtifact.kt

```diff
@@ -6,7 +6,7 @@ data class DeliveryArtifact(
 )
 
 enum class ArtifactType {
-  DEB
+  DEB, DOCKER_IMAGE
```

Comment:
could i convince you to make this `DOCKER` for YAML definition purposes? seems to fit more nicely with the `DEB` naming. Either way, I'll continue on living somehow 😄

Reply:
Strong agree

Reply:
Sure :D